src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -371,11 +371,7 @@ protected function getEnv(string $name)
         if (!$this->has($id = 'container.env_var_processors_locator')) {
             $this->set($id, new ServiceLocator([]));
         }
-        if (!$this->getEnv) {
-            $this->getEnv = new \ReflectionMethod($this, __FUNCTION__);
-            $this->getEnv->setAccessible(true);
-            $this->getEnv = $this->getEnv->getClosure($this);
-        }
+        $this->getEnv ??= \Closure::fromCallable([$this, __FUNCTION__]);
```

Comment:
psalm bug here, it's not able to compute `__FUNCTION__` correctly
let's use 'getEnv' instead